query-connector/README.md

```diff
@@ -77,7 +77,36 @@ The Query Connector will eventually require other inputs from other DIBBs servic
 
 ### Developer Documentation
 
-Can be found in [api-documentation.md](api-documentation.md).
+A Postman collection demonstrating use of the API can be found [here](https://github.com/CDCgov/dibbs-query-connector/blob/main/query-connector/src/app/assets/DIBBs_Query_Connector_API.postman_collection.json).
+
+### Query Connector Data for Query Building
+
+When initializing, the Query Connector makes the value sets associated with 200+ reportable conditions available to users tasked with building queries for their jurisdiction. To group value sets by condition and to group the conditions by type, the Query Connector obtains and organizes data from the eRSD and the VSAC in the following way:
+
+1. The Query Connector retrieves the 200+ reportable conditions from the eRSD as well as the value sets associated value set IDs.
+2. Using the value set IDs from the eRSD, the Query Connector retrieves the value sets from the VSAC, i.e., the LOINC, SNOMED, etc. codes associated with each value set ID.
+3. The Query Connector then organizes and makes the value sets organized by condition available to users who are building queries. The conditions are additionally organized by category, e.g., sexually transmitted diseases or respiratory conditions, using a mapping curated by HLN Consulting.
```

Comment:
Readability suggestion for the first sentence: "The Query Connector then organizes these value sets according to the conditions with which they're associated, making the result available to users interested in building queries."